### PR TITLE
Gannett rebase

### PR DESCRIPTION
**Pull request by mihasya**
@rabdill @fhanson I've rebased your guys' PR against what is now master - it wasn't too bad, mostly documentation conflicts.

I can now run `rake` and see a bunch of linter errors in the changes that were made in parallel to your PR. We will add some commits to this branch to fix all the linter errors, and hook up a Travis build as well.

Thanks a lot for doing this work, and sorry about the merge order snafu.

### CODE REVIEW

**Comment by rabdill** [comment]
I'm glad we could chip in!

File: files/default/yum_opsmatic_public.repo [hunk 1]
```diff
@@ -2,9 +2,9 @@
 name=opsmatic_public
 baseurl=https://packagecloud.io/opsmatic/public/el/6/$basearch
 enabled=1
-gpgcheck=0
+gpgcheck=1
 sslverify=1
-repo_gpgcheck=1
+#repo_gpgcheck=1
```
Comment (mihasya) [inline files/default/yum_opsmatic_public.repo]: @rabdill any reason for turning off repo-level metadata checks? You guys use an internal repo anyway, so this file doesn't affect you specifically, right?

**Comment by fhanson** [comment]
We did that before we had the repo mirrored internally

**Comment by mihasya** [comment]
OK, lint checks + kitchen pass locally now with flying colors. :+1: to merging this, then going to start a separate PR for making travis pass (gonna have to figure out the whole `.kitchen.cloud.yml` bit tomorrow).

@rabdill if you guys want GPG sig verification on the individual packages, all you gotta do is add the key in `files/default/9DAB4A7C.key` to your internal recipes. You can see how it's used in https://github.com/opsmatic/opsmatic-cookbook/commit/3f1c8325739757debf8944f402b23d4f8796478d

@ch7ck bump the version in `metadata.rb` and then go ahead and merge :+1: :sunglasses:

**Comment by rabdill** [comment]
Much obliged, we'll reach out if we hit any road blocks. The kitchen file
is just for specifying [how Kitchen should test the cookbook](
https://docs.chef.io/config_yml_kitchen.html), you shouldn't have any
trouble with it if you've managed to get ChefDK working somewhere.

OK, lint checks + kitchen pass locally now with flying colors. [image: :+1:]
to merging this, then going to start a separate PR for making travis pass
(gonna have to figure out the whole .kitchen.cloud.yml bit tomorrow).

@rabdill <https://github.com/rabdill> if you guys want GPG sig verification
on the individual packages, all you gotta do is add the key in
files/default/9DAB4A7C.key to your internal recipes. You can see how it's
used in 3f1c832
<https://github.com/opsmatic/opsmatic-cookbook/commit/3f1c8325739757debf8944f402b23d4f8796478d>

@ch7ck <https://github.com/ch7ck> bump the version in metadata.rb and then
go ahead and merge [image: :+1:] [image: :sunglasses:]

—
Reply to this email directly or view it on GitHub
<https://github.com/opsmatic/opsmatic-cookbook/pull/6#issuecomment-76880794>
.

**Comment by mihasya** [comment]
@rabdill yeah Kitchen is our primary internal testing mechanism, but we run it locally. I was referring specifically to the cloud bits for running the kitchen tests in Travis. We haven't tangled with that magic yet, but obviously would be cool to get that working.

**Pull request merged**